solc.js missing from downloaders directory

When I run `npm install -g https://github.com/ETCDEVTeam/emerald.git`, I get an error at the end where it's complaining about missing the `solc.js` file from the `downloaders` directory.

```
✔ sputnikvm-dev: finished installation
✔ emerald-wallet: finished installation
✔ emerald-explorer: finished installation
module.js:549
    throw err;
    ^

Error: Cannot find module '/Users/yazkhoury/.nodenv/versions/8.11.1/lib/node_modules/emerald/downloaders/solc.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

Looking inside the `downloaders` directory indeeds shows `solc.js` is missing even though it's being referenced here in this file: https://github.com/ETCDEVTeam/emerald/blob/master/package.json#L9

--- a/downloaders/solc-downloader.js
+++ b/downloaders/solc-downloader.js
@@ -1,0 +1,19 @@
+const request = require('request');
+const tar = require('tar');
+const path = require('path');
+const fs = require('fs');
+const ora = require('ora');
+const unzip = require('unzip');
+const extract = tar.x({
+  cwd: path.resolve(__dirname, '../')
+});
+
+const spinner = ora('solc: Downloading');
+spinner.start();
+
+const version = fs.readFileSync(path.resolve(__dirname, '../', '.solidity-version')).toString();
+
+const file = fs.createWriteStream(path.resolve(__dirname, '../', 'solc.js')); 
+
+const url = `https://ethereum.github.io/solc-bin/bin/soljson-${version}.js`;
+request(url).pipe(file).on('finish', () => spinner.succeed('solc: finished installation'));

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "clean": "rimraf emerald-explorer EmeraldWallet.app EmeraldWallet.dmg svmdev EmeraldWallet.AppImage EmeraldWallet.exe solc.js emerald-vault",
-    "postinstall": "npm run clean && node downloaders/sputnikvm-dev.js && node downloaders/emerald-wallet.js && node downloaders/emerald-explorer.js && node downloaders/solc.js && node downloaders/emerald-vault"
+    "postinstall": "npm run clean && node downloaders/sputnikvm-dev.js && node downloaders/emerald-wallet.js && node downloaders/emerald-explorer.js && node downloaders/solc-downloader.js && node downloaders/emerald-vault"
   },
   "bin": {
     "emerald": "./index.js"

